docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -572,8 +624,8 @@ AWS account. Continue working in the **Create custom cloud** wizard:
     provided variables.
 
     :::important
-    When running `terraform plan` and `terraform apply`, make sure you
-    add `-var-file=FILE_NAME.vars` as an option.
+    When running `terraform plan` and `terraform apply`, add `-var-file=FILE_NAME.vars`
```

Comment:
and `terraform destroy`.

Reply:
As pointed out by @jeremy-heller-aiven , we should not use `terraform destroy` in the Create a custom cloud` flow hence I'm not including it. cc @jeremy-heller-aiven